testing/MLDB-1624-more-join-test.py

```diff
@@ -1251,34 +1251,27 @@ def test_equi_join(self):
             ]
         )     
 
-    @unittest.expectedFailure
+    # MLDB-1664
```

Comment:
@nicolaskruchten This will not be as efficient as could be. Maybe we could create a story to have an efficient codepath for join on inequalities?